django/db/migrations/operations/models.py

```diff
@@ -929,6 +929,21 @@ def migration_name_fragment(self):
     def reduce(self, operation, app_label):
         if isinstance(operation, RemoveIndex) and self.index.name == operation.name:
             return []
+        if isinstance(operation, RenameIndex) and (
+            operation.old_name == self.index.name
+            or (
+                operation.old_fields
+                and set(operation.old_fields) == set(self.index.fields)
```

Comment:
This is incorrect. `RenameIndex` with `old_fields` is used only for transition from `index_together`, we should not reduce them with `AddIndex`.